### DISCUSSION TITLE
Overflow problem using combobox in AlertDialog

### DISCUSSION THREAD

**Discussion by rumeau** [twilio-labs/paste#2819] [Q&A] [answered]
Hello. I'm using AlertDialog to display a combobox and select an option.

My problem is that when I open the Combobox it hides behind the dialog footer buttons. This should be solved modifyng the overflow property of the content container (ALERT_DIALOG_BODY) of the AlertDialog but I haven't found how to do it.

This is my AlertDialog

```tsx
<AlertDialog
    heading="Set the wrapup code"
    isOpen={open}
    onConfirm={submitForm}
    onConfirmLabel="Save"
    onDismiss={cancelForm}
    onDismissLabel="Cancel"
>
    Select one of the available wrapup codes

    <Combobox items={wrapupCodes} labelText="Select a wrapup code" required onSelectedItemChange={handleWrapupCodeChange} />
</AlertDialog>
```

Here are som screenshots of the problem and how they should look like:
![image](https://user-images.githubusercontent.com/2340790/200940295-04cf0271-f384-4a49-bc5b-0dc4b94b9672.png)

![image](https://user-images.githubusercontent.com/2340790/200940472-90da8a4e-d5d1-4e79-9298-cfcd52f68972.png)

Thanks.

**Comment by SiTaggart** [comment]
Hi @rumeau 

This is a known issue that we have in our backlog to get to in the future. It's come up before https://github.com/twilio-labs/paste/discussions/1384 and here for reference.

It basically requires us detaching the list of options from the Combobox and putting them in a Portal.

We'll update you when it lands.

**Comment by shleewhite** [comment, marked as answer]
Hi @rumeau!

We have recently published a fix for this issue. Please upgrade either `@twilio-paste/combobox` to `13.0.3` or `@twilio-paste/core` to `17.0.5`.

Let me know if you continue to have issues! Thank you again for letting us know about this bug.
